sdk/nodejs/tests/runtime/langhost/run.spec.ts

```diff
@@ -1118,10 +1118,33 @@ describe("rpc", () => {
                 return { failures: undefined, ret: args };
             },
         },
+        "async_components": {
+            program: path.join(base, "064.async_components"),
+            expectResourceCount: 5,
+            registerResource: (ctx: any, dryrun: boolean, t: string, name: string, res: any, dependencies?: string[],
+                               custom?: boolean, protect?: boolean, parent?: string, provider?: string) => {
+                if (name === "c") {
```

Comment:
```suggestion
                if (name === "c" || name === "d") {
```

then kill the `else` block

Reply:
done.